perfmetrics/scripts/ml_tests/pytorch/setup_container.sh

```diff
@@ -66,7 +82,7 @@ gsutil cp start_time.txt $ARTIFACTS_BUCKET_PATH/
   # We need to run it in foreground mode to make the container running.
   echo "Running the pytorch dino model..."
   experiment=dino_experiment
-  python3 -m torch.distributed.launch \
+  torchrun \
```

Comment:
This is good.

Reply:
Yes it will work.

We already getting this warning in our current testing.
```
/[opt/conda/lib/python3.10/site-packages/torch/distributed/launch.py:178](https://cs.corp.google.com/piper///depot/google3/opt/conda/lib/python3.10/site-packages/torch/distributed/launch.py?l=178): FutureWarning: The module torch.distributed.launch is deprecated
and will be removed in future. Use torchrun.
```